### ISSUE TITLE
[20211121] Weekly AI ArXiv 만담

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#30]
- News
  -  3마녀 강림: ACL 2022 데드라인 + CVPR 2022 데드라인 + ICLR 2022 Rebuttal
  - CVPR 2022 데드라인 36시간 연장 + cmt 폭파 전날 overleaf 다운... 파란만장한 CVPR 
  - Global AI Summit (11. 25 ~ 26): http://globalaisummit.kr/home/
    - AI 학과 학회 소속 분들은 무료 등록 가능하답니다 (팝업참조)
  - 과학기술 미래인재 컨퍼런스 2021 (11. 26): https://www.hrst-conference.kr/forum
    - Thomas Frey 기조강연
  - AI학회에서 한국의 약진: EMNLP 5위 (KAIST 12개, 네이버 9개), NeurIPS 8위 (KAIST 45개, 네이버 7개)
- ArXiv
  - [Masked Autoencoders Are Scalable Vision Learners](https://arxiv.org/abs/2111.06377)
    - 이진원님 PR12 talk: https://www.youtube.com/watch?v=mtUa3AAxPNQ
    - Kaiming He, Xinlei Chen, Ross Girshick 조합!! (근데 왜 조직명은 아직 FAIR냐???)
    - 이미지 격자 패치 상당수 masking, encoding 이후에 maksed patch 가져와서 작은 decoder 써서 recon 시키기
    - 나중에 인식에  쓸때는 encoder만 가져다가 fine-tune
    - 마스킹 비율을 많이 늘려서 어려운  SSL task를 만들어 놓는 것이 중요함!
![image](https://user-images.githubusercontent.com/11782739/142749890-cc297d79-146f-40b7-af33-d9c0491e26b6.png)
![image](https://user-images.githubusercontent.com/11782739/142749923-568081a0-7892-4a34-8bb7-8b4a5128a5b8.png)
![image](https://user-images.githubusercontent.com/11782739/142750004-0ce122c0-486d-4eec-8a36-2cd81df461c3.png)
![image](https://user-images.githubusercontent.com/11782739/142750041-9bef7246-0b7e-4940-b301-462529cfed98.png)

  - [Swin Transformer V2: Scaling Up Capacity and Resolution](https://arxiv.org/abs/2111.09883)
    - ICCV 2021 Best paper 에 빛나는 Swin 저자들이 확대되어 V2를 내놓음.
    - V2의 메인테마는 Scaling-up: 모델크기 (3B), 해상도 (1536x1536), 데이터양
    - 스케일업을 위한 일부 모델 구조 변경
      - SA에서 QK를 dot-product 대신 cosine attention 으로
      - Layernorm의 위치를 pre에서 post 로
      - Windows에서 relative position bias를 log-spaced continuous 로
    - ImageNet-21k로 pretraing 후 ImageNet v2 84%, ImageNet-1k v1: 90.17 (기존 SOTA인 ViT는 JFT-3B) 
    - https://github.com/microsoft/Swin-Transformer 
![image](https://user-images.githubusercontent.com/11782739/142754250-51cdcc05-2bfc-4681-94d2-42afeb4bce8d.png)
![image](https://user-images.githubusercontent.com/11782739/142754202-c884f902-8145-440d-9754-32ac59db9d96.png)

  - [XLS-R: Self-supervised Cross-lingual Speech Representation Learning at Scale](https://arxiv.org/abs/2111.09296)
    - Meta AI에서 나온 진정한 의미의 음성인식 Scaling law 연구 (소속명 표기가 dog판 ㅋㅋ)
    - 기본 wav2vec 2.0 구조를 유지하되 parameter를 2B까지 끌어올리고
    - Pretraining으로 128개국어 50만시간 학습 후 Finetuining 으로 음성번역, 음성인식, 언어인식, 화자인식 등등등에서 놀라운 결과 (한국어는 61시간)
    - https://github.com/pytorch/fairseq/tree/main/examples/wav2vec/xlsr
    - 허깅페이스에도 https://huggingface.co/models?other=xls_r
![image](https://user-images.githubusercontent.com/11782739/142754663-97829a69-b1e7-481c-832b-fc953425e7cd.png)
![image](https://user-images.githubusercontent.com/11782739/142754670-a59001ab-40db-4465-b855-2614a17da5ce.png)
![image](https://user-images.githubusercontent.com/11782739/142754726-f27124da-f004-410a-a93d-af1bdb9a0c9d.png)
![image](https://user-images.githubusercontent.com/11782739/142754740-81c57435-dfec-4911-896f-3bfe957d4390.png)
![image](https://user-images.githubusercontent.com/11782739/142754761-eb841ffd-496f-4d5b-9abc-b042e13fde4f.png)

  - [Pre-training Graph Neural Network for Cross Domain Recommendation](https://arxiv.org/abs/2111.08268)
    - Graph constrastive SSL pretraing-finetuning 을 이용하여 cross domain 추천에 적용
    - 기존에 joint training 은 source domain bias에서 완전히 자유롭지 못함
    - 기본적으로 GIN (Graph isomorphism network) + MoCo 스타일의 InforNCE loss 활용 SSL
    - 특이한 건 Finetuning 시에 복잡한 모델 대신 MF를 쓰는 것이 더 낫다고 (with Bayesian Persolization Ranking loss)
    - Amazon-GGF를 source로 Amazon-PP를 target으로
![image](https://user-images.githubusercontent.com/11782739/142755517-ddca4f56-cf4f-4e1e-bb24-32207b9cc169.png)
![image](https://user-images.githubusercontent.com/11782739/142755613-76a8c262-78e4-4bd7-b679-e69dfe76c905.png)

  - [November 2021 Edition: Rapid Advancements in Digital Transformation in Journal of Innovation](https://www.iiconsortium.org/journal-of-innovation.htm) (by AI미래포럼 임채성 위원님) 
    - 산업 (Industrial) AI는 AI 적용을 통해 혁신적 결과물을 창출하는 산업 DX와 밀접한 관련성을 갖는다 하겠읍니다. 산업 DX(디지털 전환, 디지털 트랜스포메이션) 글로벌 선도 단체인 미국 IIC에서 Journal of Innovation  특집 아티클을 5개를 오늘 내어 놓았읍니다. 아티클 내용과 특집 링크를 아래와 같이 공유드립니다.    
    - 산업 DX Journey 대한 전체적인 틀과 개념을 보시려면: The Digital Transformation Journey in the Enterprise and its Leadership
    - 일본 도시바의 산업 DX 신 비즈니스 모델을 보시려면: Digital Transformation in Steel Inspection: Toshiba MetalSpector Customer Case Study
    - IIC에서의 Testbed, test drive와 산업 DX 관련한 사항을 보시려면: Digital Transformation in Practice: Learning from IIC Deployments (한국 기업 Inter-X 사례 포함)
    - 산업 DX 를 추진하는 기업의 신 혁신 프로세스를 보시려면: The New Innovation Process of Undergoing Digital Transformation in Industries
    - 일본 미쯔비시의 스마트팩토리에서의 DX를 향한 점진적 접근을 보시려면: A Continuous Improvement (Kaizen) Driven Approach for Realistic Digital Transformation in Smart Manufacturing

**Comment by kimyoungdo0122** [comment]
- News
    - [한국, 글로벌 AI 지수에서 종합 5위 기록...지난해보다 3계단 상승](http://www.aitimes.com/news/articleView.html?idxno=141035)
        - 미, 중, 캐나다, 영국은 1위부터 4위까지 동일하고 5~10위권에서 엎치락뒤치락
        - [글로벌 AI 지수](https://www.tortoisemedia.com/intelligence/global-ai/)
        - [글로벌 AI 인덱스로 본 한국 AI산업의 현주소 in 2020](http://www.fki.or.kr/FkiAct/Promotion/Report/View.aspx?content_id=907e2136-fa45-4901-81b7-8dbc4c00b7bc&cPage=&search_type=0&search_keyword=)  
        - Talent(28위->19위), Infrastructure(5위->6위), Operation Environment(30위->50위), Research(22위->18위), Development(3위->2위), Government Strategy(31위->7위), Commercial(25위->15위)  
![image](https://user-images.githubusercontent.com/42539153/141215622-ad350b08-e161-469c-ace0-d1fc6621ab65.png)
![image](https://user-images.githubusercontent.com/42539153/141215651-435bd261-b529-45c1-b828-26d96dac1a95.png)

    - [Alphabet Chases Wonder Drugs With DeepMind AI Spinoff Isomorphic Labs](https://singularityhub.com/2021/11/07/alphabet-chases-wonder-drugs-with-deepmind-ai-spinoff-isomorphic-labs/)
        - 딥마인드에서 Isomorphic Labs 라는 생물학, 의학, 신약 개발 등 전문 AI 연구 회사로 분할. 설립자 겸 CEO가 Demis Hassabis
        - [Introducing Isomorphic Labs](https://www.isomorphiclabs.com/blog)
        - Isomorphic이 동형(同型)이라는 뜻..
        - 굉장히 복잡하고 다이나믹하지만 생물학도 정보 처리 시스템으로 볼 수 있기 때문에 정보 과학과 생물학은 동형이라는 허사비스의 소개말.   
        - [DeepMind takes next step in robotics research](https://venturebeat.com/2021/10/31/deepmind-takes-next-step-in-robotics-research/)
        - 로보틱스 및 물리 시뮬레이션 플랫폼 MuJoCo를 인수, 오픈 소스로 개방. 로보틱스, RL, 물리 시뮬레이션 연구자들에게 기여가 될 것으로 전망
        - 딥마인드의 행보 귀추 주목 
    - [Facebook AI Research, multilingual 모델이 처음으로 bilingual 모델을 WMT에서 이겼다!](https://ai.facebook.com/blog/the-first-ever-multilingual-model-to-win-wmt-beating-out-bilingual-models/)
        - 최근 대부분의 언어모델은 Bilingual 모델이며 두 언어의 쌍으로 만들어진 데이터에서 학습됨.
        - 세계적으로 많이 사용되는 언어간 데이터에 비해 소수 언어는 데이터도 부족하며, 페이스북같은 전세계 수십 억 명이 이용하는 서비스에 bilingual 모델을 일일이 적용하기는 불가능함
        - 그래서 multilingual 모델을 연구했고 WMT에서 좋은 성능을 보임
        - [Facebook AI’s WMT21 News Translation Task Submission](https://arxiv.org/pdf/2108.03265.pdf?fbclid=IwAR1os98Sg4UxSyy_fFv4Sjv96C1ApdxMEB_tXc64Oxv2ti7v2jQpYGOaaRs)
        - 하나의 Multilingual 모델로 전세계 유저에게 서빙할 수 있다는 방향이 보인다?!

**Comment by laray5959** [comment]
- ArXiv 
    - [Rapping-Singing Voice Synthesis based on Phoneme-level Prosody Control](https://arxiv.org/abs/2111.09146)
    - [Emotional Prosody Control for Speech Generation](https://arxiv.org/abs/2111.04730)
    - [Improving Prosody for Unseen Texts in Speech Synthesis by Utilizing Linguistic Information and Noisy Data](https://arxiv.org/abs/2111.07549)

**Comment by veritas9872** [comment]
**Survival-oriented embeddings for improving accessibility to complex data structures**

![Screenshot (72)](https://user-images.githubusercontent.com/33523965/142762507-c0831abe-3204-4269-aed5-0058446c92bf.png)


Arxiv: https://arxiv.org/pdf/2110.11303.pdf

NeurIPS 2021 accept된 논문 공유해드립니다.

의료 인공지능 분야에서 딥러닝을 이용한 생존분석, 즉 어떤 현상이 언제 일어날 것인지에 대한 예측 관련 논문이 많이 연구되고 있는데 LMU Munich 영상의학과와 통계학과에서 liver tumor CT scan에서 영상을 직접 input으로 사용해 생존분석을 하는 방법을 제시했습니다.

VAE를 사용해서 영상의 feature를 추출하는 역할을 수행하도록 하고 Cox loss를 조합해서 생존 확률 예측을 하도록 했는데 이는 기존의 딥러닝 기반의 생존분석 모델은 영상 정보로부터  end-to-end 예측을 하기 어려웠기 때문에 학습 안정화를 하는데에 크게 기여했다고 생각됩니다.

![Screenshot (73)](https://user-images.githubusercontent.com/33523965/142762588-3c7dcf0f-c654-4968-9907-301bca2be1ec.png)

VAE의 latent vector를 보았을 때 실제로 tumor가 영향을 미치는 것으로 보입니다(?).

아쉬운 점은 영상 정보를 사용했음에도 불구하고 훨씬 단순한 tabular information을 사용한 방법과 성능이 크게 다르지 않다는 점입니다. 하지만 여전히 image data에서 survival prediction을 비교적 안정적으로 할 수 있는 방법을 제시해준다면 의의가 있다고 생각됩니다.

![Screenshot (74)](https://user-images.githubusercontent.com/33523965/142762810-848cc8d7-4888-4fe4-ba1e-3755625838df.png)

**Comment by hollobit** [comment]
### 뷰노 이준형님 대한영상정보학회 Emphysema challenge 수상 
- https://ksiim.org/c/202102/agenda/program
![IMG_4215](https://user-images.githubusercontent.com/998803/142761348-b1366147-1643-496b-9d10-df1f0b9a80a9.jpg)

### [단독] 정부가 ‘연구용’ 줬더니, 얼굴 영상 10만건 빼돌렸다
https://www.hani.co.kr/arti/economy/it/1019612.html

### 차세대 인공지능 연구개발에 5년간 3천18억원 투입
http://www.mtnews.net/news/view.php?idx=12233
https://www.youtube.com/watch?v=d-5neAOFP0U
- 과기정통부 '사람중심 인공지능 핵심원천기술 개발' 추진
![예타내역](https://image.zdnet.co.kr/2021/09/20/a977cbb301aeccbafe5d8b578cad30ce.png)

기후 변화의 가공할 미래 위협...생성적 적대 신경망(GANs)이 잘 보여준다
http://www.aitimes.com/news/articleView.html?idxno=141547
- 2021년 11월 11-12일 프랑스 파리 GPAI 총회 개최 
>- 장관급 대표를 포함하여 시민 사회, 학계, 산업 및 정부의 주요 국제 AI 전문가들이
>- https://www.gpai.ai/projects/climate-change-and-ai.pdf
>- https://www.newswire.ca/news-releases/artificial-intelligence-in-the-fight-against-climate-change-a-new-report-presented-at-cop26-862543770.html
>- https://thisclimatedoesnotexist.com/home

IEEE, 2022년 이후의 기술 impact에 대한 조사 
https://virtualizationreview.com/articles/2021/11/19/ieee-tech-study.aspx?m=1
![기술동향](https://virtualizationreview.com/articles/2021/11/19/~/media/ECG/virtualizationreview/Images/2021/11/ieee_tech.ashx)
>- https://transmitter.ieee.org/impact-of-technology-2022/
> 미국 의 최고 정보 책임자, 최고 기술 책임자 및 기술 리더 350명을 대상으로 한 10월 설문 조사를 기반
> "2022년에 가장 중요한 기술은 - 21%가 AI와 머신 러닝,
